src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -494,9 +497,28 @@ compiler.err.same.binary.name=\
 compiler.err.duplicate.case.label=\
     duplicate case label
 
+compiler.err.pattern.dominated=\
+    this pattern is dominated by a preceding pattern
+
 compiler.err.duplicate.default.label=\
     duplicate default label
 
+compiler.err.duplicate.total.pattern=\
+    duplicate total pattern
+
+compiler.err.total.pattern.and.default=\
+    switch has both a total pattern and a default label
+
+# 0: type, 1: type
+compiler.err.constant.label.not.compatible=\
+    constant label of type {0} is not compatible with switch selector type {1}
+
+compiler.err.flows.through.to.pattern=\
+    illegal fall-through to a pattern
+
+compiler.err.multiple.patterns=\
+    multiple pattern declarations
```

Comment:
This text is very terse and doesn't really say what's wrong with the code. I think the point here is that we don't want code like this:

```
case String s, Integer i: ...
```
because this is morally:

```
case String s:
case Integer i: ...
```
which is, again, fall-through to a pattern. Maybe, just reusing the same "fall-through" message would be ok here?